L1Trigger/TrackFindingTracklet/python/Customize_cff.py

```diff
@@ -0,0 +1,25 @@
+import FWCore.ParameterSet.Config as cms
+
+def newKFConfig(process):
+  process.TTTracksFromTrackletEmulation.Fakefit = True
+
+def fwConfig(process):
+  newKFConfig(process)
+  process.TrackTriggerSetup.Firmware.FreqBE = 240
+  process.TTTracksFromTrackletEmulation.RemovalType = ""
+  process.TTTracksFromTrackletEmulation.DoMultipleMatches = False
+  process.TTTracksFromTrackletEmulation.EmulateTB = True
+  process.ChannelAssignment.UseDuplicateRemoval = False
+
+def reducedConfig(process):
+  fwConfig(process)
+  process.TrackTriggerSetup.KalmanFilter.NumWorker = 1
+  process.ChannelAssignment.SeedTypes = cms.vstring( "L1L2" )
+  process.ChannelAssignment.SeedTypesSeedLayers = cms.PSet( L1L2 = cms.vint32( 1,  2 ) )
+  process.ChannelAssignment.SeedTypesProjectionLayers = cms.PSet( L1L2 = cms.vint32(  3,  4,  5,  6 ) )
+  process.ChannelAssignment.MaxNumProjectionLayers = 4
+  process.ChannelAssignment.IRChannelsIn = cms.vint32( 0, 1, 25, 2, 26, 4, 28, 5, 29, 6, 30, 7, 31, 8, 32, 9, 33 )
```

Comment:
Clarify how the user should get the correct values of IRChannelsIn for other reduced configs. Where did you get these numbers from?

Reply:
I think the code at https://github.com/cms-L1TK/cmssw/blob/tschuh_customize/L1Trigger/TrackFindingTracklet/plugins/L1FPGATrackProducer.cc#L454 could be made more obvious by not converting directly from TFP input channel to DTC name, but instead converting via the DTC ID. e.g.:
int dtcId = setup_.dtcId(region, channel);
int slot = setup_.dtcId(dtcId);
string dtcName = dtcbasenames[slot];
if (not setup_.side(dtcId))  dtcname = "neg" + dtcname;

Though I'm not sure we can get the A_ & B_ flags via dtcId?

This change would also ensure that  all code that relates TFP channel to DTC ID is only in class Setup.

Reply:
On related note, https://github.com/cms-L1TK/cmssw/blob/tschuh_customize/L1Trigger/TrackTrigger/python/ProducerSetup_cfi.py#L136 suggests that each DTC crate (=nonant) has three 10Gb PS slots, whereas  https://github.com/cms-L1TK/cmssw/blob/tschuh_customize/L1Trigger/TrackFindingTracklet/plugins/L1FPGATrackProducer.cc#L455 suggests there are 4. Is this inconsistent?

Reply:
one detector nonant consists out of 2 ATCA crates (each having 12 slots). One crate contains 3 PS10G, 3PS5G and 12 2S5G.

Reply:
that was the info I had some years ago, I wonder if that has changed in the past

Reply:
But yes, my code and tracklet code are inconsistent regarding connection speed of one PS DTC per crate, the connection speed however is only used by DTC emulator to get truncation right. So it is more a naming thing.

Reply:
I am not a big fan of parsing txt files, especially txt files which are not part of the repo and even more especially when the strings are misleading as PS10G_4.

Reply:
> Add comment to start of HybridTracksNewKF_cfg.py explaining what it is for. i.e. Why one wouldn't just run L1TrackNtupleMaker_cfg.py.

done